package.json

```diff
@@ -16,6 +16,7 @@
     "core-js": "^3.6.5",
     "dns": "^0.2.2",
     "postcss": "^7.0.36",
+    "pre-commit": "^1.2.2",
```

Comment:
What was the need for adding this?

Reply:
Pre-commit wasn't added to this project locally.